src/test/test_address_set.c

```diff
@@ -61,7 +61,7 @@ test_contains(void *arg)
   tor_addr_parse(&addr_v6, "1:2:3:4::");
   tor_addr_t addr_v4;
   tor_addr_parse(&addr_v4, "42.42.42.42");
-  uint32_t ipv4h = tor_addr_to_ipv4h(&addr_v4);
+  //uint32_t ipv4h = tor_addr_to_ipv4h(&addr_v4);
```

Comment:
why it is commented?

Reply:
This is the commit:
https://github.com/anyone-protocol/ator-protocol/pull/96/commits/a3a4ffce85c9fc4374c94e92e5753630ae3d47e1

I just added the commits, that Guardian Project did and weren't merged into Tor to add JNI support. I didn't test if this commit is strictly necessary.

Reply:
Ok, I reverted the patch to disable the tests. To integrate that patch probably was a bad idea, since this is not just the Android version. (As the Guardian Project Tor fork actually is.)

Can we move on now?